package.json

```diff
@@ -19,7 +19,10 @@
     "build": "tsc --project tsconfig.build.json  && npm run assets:copy",
     "start": "npm run build && cd dist && node ./index.js",
     "assets:copy": "copyfiles -f ./config/* ./dist/config && copyfiles -f ./openapi3.yaml ./dist/ && copyfiles ./package.json dist",
-    "clean": "rimraf dist"
+    "clean": "rimraf dist",
+    "docker:build": "docker build -f ./Dockerfile --rm -t vector-tiles-rasterizer:v1 .",
+    "docker:run": "docker run --rm --network host -it --name vtr vector-tiles-rasterizer:v1",
```

Comment:
same

Reply:
the docker run command got a little more complex now that there is a need to mount a style file and provide `APP_STYLE_FILE_PATH` as its path. so I've removed `docker:run` script, unless you think otherwise